cmd/deploy/proxy.go

```diff
@@ -94,9 +95,9 @@ func NewProxy(kubeconfig *KubeConfig) (*Proxy, error) {
 	s := &http.Server{
 		Addr:         fmt.Sprintf(":%d", port),
 		Handler:      handler,
-		ReadTimeout:  5 * time.Second,
-		WriteTimeout: 10 * time.Second,
-		IdleTimeout:  120 * time.Second,
+		ReadTimeout:  okteto.GetKubernetesTimeout() * time.Second,
```

Comment:
Do we need to have `* time.Second`. I would add the `* time.Second` on `okteto.GetKubernetesTimeout()` function. I think it's cleaner
```suggestion
		ReadTimeout:  okteto.GetKubernetesTimeout(),
```

Reply:
@jLopezbarb @jpf-okteto updated!